bot/tests/conftest.py

```diff
@@ -331,3 +331,40 @@ def mock_taskcluster():
     taskcluster_config.options = {
         'rootUrl': 'http://taskcluster.test',
     }
+
+
+def covdir_report(codecov):
```

Comment:
This is a good workaround for now, but maybe we should in the future totally remove the mock reports in the old format.

Reply:
Yes, but i wanted to show that the data did not change in that diff.